birdhouse/components/monitoring/config/magpie/config.yml.template

```diff
@@ -52,3 +52,27 @@ permissions:
     permission: write
     group: administrators
     action: create
+  - service: grafana
```

Comment:
Oh, I think I forgot to explain that the `monitoring` group is there so existing magpie users can be added to that group, and then we do not have to share the `admin` user account.

Reply:
That makes sense. I just meant that we could add the `admin` user to the `monitoring` group as well. That way we wouldn't have to have the list of permissions for the `administrators` group and the `monitoring` group duplicated in this file.

Reply:
Further explained in this commit 0b6d954a283ea0d99871e5fd17a4a4ff6d800af4

I would keep the `administrators` group as-is.  Otherwise, it's tedious to keep in sync the member list of both `administrators` and `monitoring`.

If we promote someone to be admin already, he would need to only be added to the `administrators` group and that person would have access to everything.

If we only want someone to access the monitoring WebUI but do not want the person to be full blown admin, then we would add to the `monitoring` group.

I agree the current duplicated syntax is ugly.  I think the proper fix is for magpie to accept a list of groups instead of one single group in that config file.  A list of permissions as well, since currently `read` and `write` permission require 2 blocks.

Reply:
Given that `admin` is part of `administrators`, it doesn't matter whether it is part of `monitoring` group as well or not. It already as full access from being an administrator. For the same reason, all the duplicated permissions with `group: administrators` are redundant.